docs/src/Touchable.doc.js

```diff
@@ -48,7 +48,8 @@ card(
       },
       {
         name: 'onTouch',
-        type: '({ event: SyntheticMouseEvent<> }) => void',
+        type:
+          '({ event: SyntheticEvent<> | SyntheticKeyboardEvent<> }) => void',
         required: true,
```

Comment:
Doc states that this prop is required but it was (flow) annotated as optional. Some internal call sites currently treat this as optional.

Reply:
I'll leave the doc update to another PR since it's not directly relevant to this change.